docs/release-notes.rst

```diff
@@ -6,6 +6,97 @@
+      Sign-On will be used for these users.
+
+**New Features**
+
+-  Kubernetes: In the enterprise edition, add the ability to set up the Determined master service on
+   one Kubernetes cluster and manage workloads across different Kubernetes clusters. Additional
+   non-default resource managers and resource pools are configured under the master configuration
+   options ``additional_resource_managers`` and ``resource_pools`` (additional resource managers are
+   required to have at least one resource pool defined). Additional resource managers and their
+   resource pools must have unique names. For more information, visit :ref:`master configuration
+   <master-config-reference>`. Support for notebooks and other workloads that require proxying is
+   under development.
+
+-  WebUI: In the enterprise edition, add the ability to view resource manager name for resource
+   pools.
+
+-  API/CLI/WebUI: In the enterprise edition, route any requests to resource pools not defined in the
+   master configuration to the default resource manager, not any additional resource manager, if
+   defined.
+
+-  Configuration: In the enterprise edition, add a ``name`` and ``metadata`` field to the resource
+   manager section in the master configuration. Add an ``additional_resource_managers`` section that
+   follows the ``resource_manager`` and ``resource_pool`` configuration pattern.
+
+**Improvements**
+
+-  Configuration: The master configuration parameter ``observability.enable_prometheus`` now
+   defaults to ``true``. Consequently, Prometheus endpoints are enabled by default, which does not
+   affect clusters that do not use Prometheus.
+
+**Bug Fixes**
+
+-  Since version 0.27.0, the selection functionality in the Experiments page had a bug where it
+   attempts to perform actions on a larger set of experiments than selected on the screen. For
+   instance, if you try to move all experiments in one project into another using Select All ->
+   Actions -> Move, it may inadvertently include all experiments from the project also from other
+   projects you have access to edit. We urge all users on the affected versions to upgrade as soon
+   as possible.
+
+   -  There is a risk of data loss if attempting to delete a set of experiments delete and it
+      inadvertently deletes a larger set than intended.
+
+   -  For RBAC users, thereis a risk of permissions leak if moving experiments from one project to
+      another includes experiments from other workspaces.
+
+   -  The behavior is true for all bulk actions (delete, move, archive, unarchive, resume, pause,
+      kill, stop, and view in TensorBoard).
+
+   -  We strongly advise refraining from using the Experiment table in the project view to take any
+      actions. As a workaround, utilize the Trial View in the Web UI for actions on a single trial,
+      or use the CLI for affected actions as an alternative.
```

Comment:
```suggestion
-  Starting in version 0.27.0, the selection functionality in the experiments list page has had a
   bug where it attempted to perform actions on a larger set of experiments than shown on the
   screen. For instance, if you tried to move all experiments in one project into another using
   Select All -> Actions -> Move, it may also include experiments from other projects you had access
   to edit. We urge all users on the affected versions to upgrade as soon as possible.

   -  There is a risk of data loss if attempting to delete multiple experiments.

   -  For RBAC users, there is a risk of permissions leak if moving experiments from one project to
      another includes experiments from other workspaces.

   -  The behavior is true for all bulk actions (delete, move, archive, unarchive, resume, pause,
      kill, stop, and view in TensorBoard).

   -  We strongly advise refraining from using the experiment table in the project view to take any
      actions. As a workaround, use the trial view in the WebUI for actions on a single trial or the
      CLI for bulk actions.
```

Reply:
sorry - I'll use Tara's suggestions

Reply:
No problem, they look pretty solid, though let me take a moment to tweak just a bit.